datasets/Met-Office-Regional-average-climate-observations-UK-annual-rainfall-2021/main.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd 
+from gssutils import *
+from csvcubed.models.cube.qb.catalog import CatalogMetadata
+from datetime import datetime
+
+df = pd.read_csv("raw.csv")
+
+df['Year'] = df['Year'].astype(str).astype(int)
+
+df.to_csv('observations.csv', index=False)
+catalog_metadata = CatalogMetadata(
+    title = "Regional average climate observations uk annual rainfall 2021",
+    creator_uri = "https://www.gov.uk/government/organisations/the-meteorological-office",
+    publisher_uri = "https://www.gov.uk/government/organisations/met-office",
+    description = "Priority dataset for Climate Change Platform project."
```

Comment:
remove the description (line 15) for now as this should describe what the data is representing. We will need to obtain a proper one from either the publishers or try to find one ourselves.